src/main/java/net/sf/jabref/logic/importer/fetcher/MrDLibFetcher.java

```diff
@@ -103,17 +111,23 @@ private String makeServerRequest(String query) {
         return response;
     }
 
+    /**
+     * Constructs the query based on title of the bibentry. Adds statistical stuff to the url.
+     * @param query: the title of the bib entry.
+     * @return
+     */
     private String constructQuery(String query) {
         StringBuffer queryBuffer = new StringBuffer();
-        queryBuffer.append("https://api-dev.mr-dlib.org/v1/documents/");
-        queryBuffer.append(query);
+        //queryBuffer.append("https://api-dev.mr-dlib.org/v1/documents/");
```

Comment:
Can you move that to a test case somehow?

You should use the method `getURLForID` instead of `constructQuery` as this seems the functionality you need. Don't you? - If not, please document the difference to `getURLForID` in the method comemnt